create_SQLite_table.py

```diff
@@ -0,0 +1,63 @@
+import sqlite3
+
+sqlite_file = 'reactors'    # name of the sqlite database file
```

Comment:
I'd usually call such a file "reactors.sqlite" so that the file extension makes it clear what kind of file it is.